articles/libraries/auth0-android/passwordless.md

```diff
@@ -11,13 +11,28 @@ useCase: enable-mobile-auth
 ---
 # Auth0.Android Passwordless Authentication
 
-<%= include('../../_includes/_native_passwordless_warning') %>
-
 <dfn data-key="passwordless">Passwordless</dfn> can be done via email or via SMS, and either by sending the user a code, or sending them a link which contains a code. All of these methods of Passwordless authentication will require two steps - requesting the code, and then inputting the code for verification. 
 
-Note that Passwordless authentication **cannot be used** with the [OIDC Conformant Mode](/api-auth/intro) enabled.
+## Configure Auth0 and the Android SDK
+
+### Enable the Passwordless OTP Grant for the Application
+
+In order to be able to use the Passwordless API from a Native client, you first need to enable the Passwordless OTP grant for your application in **Dashboard > Applications > (YOUR APPLICATION) > Settings > Advanced Settings > Grant Types**.
 
-## 1. Request the code
+### Initialize the Android SDK 
+
+Using the Passwordless API requires setting using the Auth0 Android SDK version 1.20 or higher, configured to work in OIDC conformant mode. This can be achieved by setting the `ODICConformant` property to `true`:
```

Comment:
typos

```suggestion
Using the Passwordless API requires the Auth0 Android SDK version 1.20 or higher, configured to work in OIDC conformant mode. This can be achieved by setting the `OIDCConformant` property to `true`:
```